app/src/utils/handlebarsUtils.ts

```diff
@@ -0,0 +1,69 @@
+import Handlebars, { HelperOptions } from 'handlebars';
+
+export const useHandlebars = () => {
+  /**
+   * This functions allows us to conditionally check values
+   *
+   */
+  const applyConditionalChecks = () => {
+    Handlebars.registerHelper('ifCond', (v1, operator, v2, options: HelperOptions) => {
+      switch (operator) {
+        case '==':
+          return v1 === v2 ? options.fn(this) : options.inverse(this);
+        case '===':
+          return v1 === v2 ? options.fn(this) : options.inverse(this);
+        case '!=':
+          return v1 !== v2 ? options.fn(this) : options.inverse(this);
+        case '!==':
+          return v1 !== v2 ? options.fn(this) : options.inverse(this);
+        case '<':
+          return v1 < v2 ? options.fn(this) : options.inverse(this);
+        case '<=':
+          return v1 <= v2 ? options.fn(this) : options.inverse(this);
+        case '>':
+          return v1 > v2 ? options.fn(this) : options.inverse(this);
+        case '>=':
+          return v1 >= v2 ? options.fn(this) : options.inverse(this);
+        case '&&':
+          return v1 && v2 ? options.fn(this) : options.inverse(this);
+        case '||':
+          return v1 || v2 ? options.fn(this) : options.inverse(this);
+        default:
+          return options.inverse(this);
+      }
+    });
+  };
+
+  /**
+   * This function converts a rawTemplate to a template
+   *
+   * @param {string} template
+   * @return {*}  {HandlebarsTemplateDelegate}
+   */
+  const compileFromRawTemplate = (template: string): HandlebarsTemplateDelegate => {
+    applyConditionalChecks();
+    return Handlebars.compile(template);
+  };
+
+  /**
+   * This function converts a precompiled template
+   * see Readme/handlebars.md for more information
+   *
+   * @param {TemplateSpecification} preCompiledtemplate
+   * @return {*}  {HandlebarsTemplateDelegate}
+   */
+  const compileFromPrecompiledTemplate = (preCompiledtemplate: TemplateSpecification): HandlebarsTemplateDelegate => {
+    const encodedHandlebarsFunction = `(handlebars) => handlebars.template(${preCompiledtemplate})`;
+    const handlebarsFunction = eval(encodedHandlebarsFunction);
```

Comment:
Can you add an ignore comment for this:

`// eslint-disable-next-line no-eval`

Reply:
We might also want to add some comments in general to this explaining why we are using eval in the first place